server.js

```diff
@@ -51,28 +51,42 @@ const categoriesUrl = `${apiUrl}categories?per_page=100`
 
 // functions//
 
-// date parser
-const datePars = function(postData,categoryData){ // ask how to implement this function and fix the post data
-	for(var i = 0; i < postData.length;i++){
-	const parsedDate = new Date(postData[i].date),
-	day = parsedDate.getDate(),
-	options = { month: "short" }, // De maand moet kort geschreven zijn
-	month = Intl.DateTimeFormat("nl-NL", options).format(parsedDate.getMonth() + 1),
-	newDate = day + ''+ month;
-	postData[i].date = newDate;
-	categoryData[i].date = newDate;
-	postData[i].author = userData.find(user => user.id === postData[i].author);
-	console.log('date parser works');
-	return(postData)
-}}
-
-
-const formatPostMeta = (postData) => {
-	const options = { day: 'numeric', month: 'long' };
-	const formattedDate = new Date(post.date_gmt).toLocaleDateString('nl-NL', options);
-	const estimatedReadingTime = post.yoast_head_json.twitter_misc["Geschatte leestijd"].replace("minuten", "min");
-	return `${formattedDate} - ${post.yoast_head_json.author} - ${estimatedReadingTime}`;
-};
+// metaParse
+function metaParse(postsData) {
+    postsData.forEach(postData => {
+        const postDate = new Date(postData.date_gmt);
+        
+        const dateOptions = { day: 'numeric', month: 'long' };
+        const timeOptions = { hour: '2-digit', minute: '2-digit' };
+        
+        const formattedDate = postDate.toLocaleDateString('nl-NL', dateOptions);
+        const formattedTime = postDate.toLocaleTimeString('nl-NL', timeOptions);
+
+		const author = postData.yoast_head_json?.twitter_misc?.["Geschreven door"] ?? "N/A";
+
```

Comment:
Idk what the indentation is doing here, but it's not there in my VS code.

Reply:
yeah it can be a random glitch